docs/Components/Powertrain/SimpleMotor.md

```diff
@@ -0,0 +1,57 @@
+# Simple Motor
+
+Simple Motor class is based on the class called "attatched component", and written over the Motor class. For the fact that it is an attatched component, it is needed to set a parent for it, and just like on attached component, all information from the parent will be taken in count on the children methods.
```

Comment:
...based on the Motor class, which inherits from the AttachedComponent class, ...